app/app.go

```diff
@@ -352,6 +360,21 @@ func NewOsmosisApp(
 
 	app.sm.RegisterStoreDecoders()
 
+	// initialize lanes + mempool
+	mevLane, defaultLane, freeLane := CreateLanes(app, txConfig)
+
+	// create the mempool
+	lanedMempool, err := block.NewLanedMempool(
+		app.Logger(),
+		[]block.Lane{mevLane, defaultLane, freeLane},
+	)
```

Comment:
the default lane is going to match all transactions that are in the free lane, so i recommend we switch to mev, free, default